src/apirequest.ts

```diff
@@ -74,7 +74,7 @@ async function createAPIRequestAsync<T>(parameters: APIRequestParams) {
   // Create a new params object so it can no longer be modified from outside
   // code Also support global and per-client params, but allow them to be
   // overriden per-request
-  const topOptions = parameters.context.google
+  const topOptions = parameters.context.google._options
```

Comment:
Thank you!  Could you add a double layer of protection here?

```suggestion
  const topOptions = (parameters.context.google && parameters.context.google._options) 
```

Reply:
do you also want to validate params. as so: 
  const topOptions = (parameters.context.google && parameters.context.google._options && parameters.context.google._options.params)

Reply:
I don't think that's strictly required here.  To be honest I'm surprised there are situations where any of these are null/undefined, but it doesn't hurt to be careful :)